src/main/java/org/mule/module/xml/internal/operation/XsltOperation.java

```diff
@@ -81,6 +83,14 @@ public String xsltTransform(@Content(primary = true) InputStream content,
     return withTransformer(xslt, transformer -> {
       bindParameters(transformer, contextProperties);
 
+      // TODO: Question: Should I put this in try-finally block to deal with the case something fails in between,
```

Comment:
remove comment. And per the question, I don't think this makes much sense. It looks like all the transformers end up using the same resolver anyway. Plus keep in mind these are pooled. So next time a transformer is used, the resolver gets overwritten anyway